templates/download/shared/_footer.html

```diff
@@ -8,11 +8,15 @@ <h2>Get started with Ubuntu today</h2>
       <hr class="p-rule" />
       <div class="row">
         <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Professional support for Ubuntu</h3>
-          <p>Get professional support for Ubuntu from Canonical. We help organisations around the world to manage their Ubuntu cloud, server and desktop deployments.</p>
+          <h3 class="p-heading--5">Get comprehensive security coverage</h3>
```

Comment:
A small update from the [copydoc](https://docs.google.com/document/d/1VhmFVE3dK81mE3zcC9FVA6f5SGJ-DHan7WWlrlfRcvo/edit?tab=t.0#heading=h.5uch6alr3qy6)
```suggestion
          <h3 class="p-heading--5">Comprehensive security coverage with Ubuntu Pro</h3>
```